.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    commit-message:
+      # Prefix all commit messages with "deps: ", which should be
+      # accepted as a conventional commit and trigger release-please
```

Comment:
Why is `release-please` mentioned here?

Reply:
It was copy-pasted from rust-bio. It triggers a release there. Does not do anything here. Edited the comment.